aim/sdk/adapters/prophet.py

```diff
@@ -0,0 +1,76 @@
+from sys import stdout
+from typing import Any, Dict, Optional, TypeVar
+
+from aim import Run
+from aim.ext.resource import DEFAULT_SYSTEM_TRACKING_INT
+
+
+Prophet = TypeVar("Prophet")
+
+
+class AimLogger:
+    def __init__(
+        self,
+        prophet_model: Prophet,
+        repo: Optional[str] = None,
+        experiment: Optional[str] = None,
+        system_tracking_interval: int = DEFAULT_SYSTEM_TRACKING_INT,
+        log_system_params: bool = True,
+        log_cout=stdout,
+    ):
+        """
+        AimLogger for Prophet models.
+        Prophet doesn't have a callback system and isn't trained iteratively.
+        Thus, only the hyperparameters and user-provided metrics are logged.
+        """
+        self._repo_path = repo
+        self._experiment = experiment
+        self._system_tracking_interval = system_tracking_interval
+        self._log_system_params = log_system_params
+        self._run = None
+        self._run_hash = None
+        self._log_cout = log_cout
```

Comment:
Seems the `self._log_cout` member is not used? Should it be removed?

Reply:
Hi Albert jan, yup you're right I'll remove the log_cout it in a sec. Will also add to the changelog.

Reply:
Hey Albert jan, just addressed your comments in the latest commits :)